CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+This project is open to contributors, and we'd like to make it as easy as possible to share your ideas, and your code.
+
+Firstly, no contribution is too small. If you see something that's not right, please feel free to submit a spontaneous PR. It will very likely be reviewed same day.
+
+## Where to start
+
+If you don't know where to start, have a look at the list of [issues](https://github.com/tomarrell/lbadd/issues). If you find something interesting, feel free to say so in a comment, and you can be assigned to the issue.
+
+If someone is already assigned to an issue, feel free to mention that you're interested as well. Most issues can likely be broken down into more fine-grained tasks for newcomers to contribute to.
+
+If you don't find something in the issues, don't worry! There is **plenty** that is *likely not listed there*. If you think you have a clear idea about something that should be done, feel free to open an issue yourself and start a discussion. A maintainer will get back to you, and most likely assign you to the task or clarify some requirements beforehand.
+
+## Project Setup
+
+In order to begin contributing, please fork the project to your personal Github account. You can then clone the project to your machine, and begin work on a new branch.
+
+You'll need to have installed:
+
+- Go compiler `version >=1.13`
+- golint `master // TODO`
+- errcheck `master // TODO`
+- gosec `master // TODO`
+
```

Comment:
Once the project is more stable, `staticcheck` is also required, right?

Reply:
Yes that's correct. I'll probably look a little further into using `golangci-lint`, and whether it would be beneficial in comparison to our current setup. But for now, things are fine.

Reply:
Didn't get it to work for me last time I used it, but maybe this time it works.